sys/vm/swap_pager.c

```diff
@@ -2393,12 +2393,13 @@ swp_pager_meta_transfer(vm_object_t srcobject, vm_object_t dstobject,
 				swp_pager_meta_build(dstobject,
 				    sb->p + i - offset, sb->d[i], false);
 				VM_OBJECT_WLOCK(srcobject);
+			} else if (blk != SWAPBLK_NONE)
+				swp_pager_update_freerange(&range, sb->d[i]);
 #if __has_feature(capabilities)
+			if (blk != SWAPBLK_NONE)
```

Comment:
I think this `blk != SWAPBLK` predicate is wrong. That's testing whether the destination object already had a swap block assigned to the current pindex, but we don't care about that here.

Reply:
So just do it unconditionally?

Reply:
Yes. If there is no swap block at this pindex in the source object, we won't get to this point. If there is a swap block, we always want to copy tags.

I think it's wrong that we potentially do that after dropping and re-acquiring the object lock, but this bug is (well, was) in the upstream code and has since been fixed, so there will be a future merge conflict here to deal with.